etl/firefox_legacy.py

```diff
@@ -0,0 +1,74 @@
+import json
+import os
+
+import requests
+
+from .search import create_metrics_search_js
+
+OUTPUT_DIRECTORY = os.path.join("public", "data")
+FUNCTIONS_DIRECTORY = ".netlify"
+PROBES_URL = "https://probeinfo.telemetry.mozilla.org/firefox/all/main/all_probes"
+
+# a bit of a hack, we would ideally make this public data. this data is already implicitly
+# public via the probe search service, so putting this here. probably best to make this
+# a public data set produced by bigquery-etl in the long run though
+PROBE_RECORDED_IN_PROCESSES_URL = "https://sql.telemetry.mozilla.org/api/queries/83152/results.json?api_key=Z1WhC2sRmLXmSwZyUwhIOFQQQAUK3JvGNLZgtZGW"  # noqa
```

Comment:
This should be replaced with a link to the public data json once https://github.com/mozilla/bigquery-etl/pull/2555 lands